velox/functions/sparksql/String.h

```diff
@@ -1019,19 +1015,40 @@ struct ConvFunction {
     // When toBase is negative, converts to signed value. Otherwise, converts to
     // unsigned value. Overflow is allowed, consistent with Spark.
     if (toBase < 0) {
+      // This flag is used to make sure when we calculate the resultSize in
```

Comment:
This logic is very confusing. Let's not mix it with the rest of the code. Perhaps, extract a helper function to compute result size.

Reply:
I made two functions for signed and unsigned case, hope it is clear now